src/components/include/cmap.h

```diff
@@ -121,43 +146,42 @@ cmap_add(cmap_t *m, void *val)
 	/* no free slots? Create more! */
 	if (free == -1) {
 		long lower, upper;
-		if (val != CVECT_INIT_VAL && 
-		    0 > cvect_add_id(m->data, val, cmap_to_vect_id(m->id_boundary))) return -1;	
-		free = lower = m->id_boundary;
+		if (val != CVECT_INIT_VAL && 0 > cvect_add_id(m->data, val, cmap_to_vect_id(m->id_boundary))) return -1;
+		free = lower   = m->id_boundary;
```

Comment:
It wants to align the "real" assignment on the end. It sucks, but idk how to fix it.